src/Ninja/G1CP/Content/Misc/world.d

```diff
@@ -257,3 +257,19 @@ func int G1CP_GetWorldTime() {
 
     return hour * 100 + min;
 };
+
+/*
+ * Find VOB in the world by its exact position and move it to another position. Returns true on success.
+ *
+ * Simplifies fix template for "changeVobLocation"
+ */
+func int G1CP_ChangeVobLocation(var int originalPosPtr, var int correctedPosPtr, var int callback) {
```

Comment:
The parameter `callback` should be of type `func` to allow supplying `NOFUNC` when no callback is desired, see the find-functions in the file above.